src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/DefaultCompressPlugin.java

```diff
@@ -92,16 +92,29 @@ public void configure(Map<String, String> config) {
         if (level != null) {
             switch (level) {
                 case LEVEL_0:
+                    System.err.println("warning - the \"0\" value for --compress has been deprecated and " +
```

Comment:
I expect this be a localized message, e.g. add to plugins.properties.   You can call AbstractPlugin::getMessage` to get the message from the resource bundle.  This one should be consistent with the existing convention but unfortunately the existing warning messages are not consistent - prefixing with "Warning:" or "WARNING:".

Reply:
Thanks. I note that there is a non-localized warning done this way elsewhere in jlink. May need to make a bug to localize this.

Reply:
I prefer this one should use the localized message.  The other ones should be followed up in a separate bug as they are unrelated to this one.

Reply:
Agree! Sorry, got my words mixed up. The existing non-localized behavior should put into another bug.